dev/fattest.simplicity/src/componenttest/containers/ArtifactoryImageNameSubstitutor.java

```diff
@@ -79,8 +79,9 @@ private static boolean isSyntheticImage(DockerImageName dockerImage) {
 
     static String getPrivateRegistry() {
         String artifactoryServer = System.getProperty(artifactoryRegistryKey);
-        if (artifactoryServer == null || artifactoryServer.isEmpty() || artifactoryServer.startsWith("${"))
-            throw new IllegalStateException("No private registry configured. System property '" + artifactoryRegistryKey + "' was: " + artifactoryServer);
+        if (artifactoryServer == null || artifactoryServer.isEmpty() || artifactoryServer.startsWith("${") || artifactoryServer.equals("null"))
+            throw new IllegalStateException("No private registry configured. System property '" + artifactoryRegistryKey + "' was: " + artifactoryServer + "  "
+                                            + "Ensure artifactory properties are set in gradle.startup.properties");
```

Comment:
Was there a reason this line was removed?

Reply:
Unless I'm missing something nothing was removed here? I added an additional check for the string "null" because I hit that from not having artifactory properties in my gradle.startup.properties file, and then added the message about checking gradle.startup.properties in the Exception.